### DISCUSSION TITLE
Welcome to Community Discussions!

### DISCUSSION THREAD

**Discussion by de1mat** [Equipd/Bible#1] [Announcements]
## 👋 Welcome!

I started setting this up to replace the older forums a while ago, but wasn't convinced enough to officially switch across. But now is the time. The older forums were getting harder to maintain, and over time functionality has started breaking. I will start shuffling existing issues and other content across. Please create a GitHub account if you don't have one, and feel free to:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

Thanks for your ongoing support of the [Equipd Bible app](https://www.equipd.me/) 🤗

**Comment by cienypico** [comment]
Thanks Matt - another year rapidly coming to a close - looking forward to being able to receive 2025 schedules in due course

**Reply by de1mat** [reply]
Thanks @cienypico, will create them in the near future. Thanks for checking in.

**Note:** I am locking comments on this thread and will delete this comment in the near future to keep this "Welcome" thread on topic. Please feel free to start a new discussion related to any ideas or feedback. Thanks for taking the time to share your thoughts 🤗

**Reply by de1mat** [reply]
The initial English schedules for 2025 are [in place now](https://www.equipd.me/forums-archived/forums/schedules/).

Easiest installed from inside the app. Go to the schedules tab, click on the settings icon, and click the link to download new schedules. This will open the forums and when you locate and click the ZIP file it will automatically install it for you.

**Comment by BRANthaD0N** [comment]
Would there be any chance for dark mode?

**Reply by de1mat** [reply]
I hope to implement this in the future yes. I have some bigger plans for this app, just need to get a few ducks in a row.

**Note:** I am locking comments on this thread and will delete this comment in the near future to keep this "Welcome" thread on topic. Please feel free to start a new discussion related to any ideas or feedback. Thanks for taking the time to share your thoughts 🤗
